dali/pipeline/operators/reader/loader/loader.h

```diff
@@ -59,7 +59,8 @@ class Loader {
       read_ahead_(options.GetArgument<bool>("read_ahead")),
       stick_to_shard_(options.GetArgument<bool>("stick_to_shard")),
       device_id_(options.GetArgument<int>("device_id")),
-      skip_cached_images_(options.GetArgument<bool>("skip_cached_images")) {
+      skip_cached_images_(options.GetArgument<bool>("skip_cached_images")),
+      lazy_(options.GetArgument<bool>("lazy")) {
```

Comment:
Knitpick: maybe something more descriptive (`lazy_init` or something)

Reply:
Resolve in #746